### PR TITLE
#6 - github actions

### LINKED ISSUE #6: Add github actions

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #6 
Set github actions

### CODE REVIEW

File: .github/workflows/release-cli.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      - name: Build
+        run: mvn clean package -Pcli
+      - name: Upload binaries to release
+        uses: svenstaro/upload-release-action@v1-release
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: target/debian-adapter.jar
+          asset_name: debian-adapter.jar
+          tag: ${{ github.ref }}
+          overwrite: true
```
Comment (olegmoz) [inline .github/workflows/release-cli.yml:21]: @olenagerasimova not sure if we really need this action. I see there is similar one in `rpm-adapter`, though it's unclear why it was added

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see one comment

**Comment by olenagerasimova** [comment]
@olegmoz thanks, this action is not required here, removed it

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, good to merge

**Pull request merged**